app/src/main/java/com/example/sabonit/CategoriesActivity.java

```diff
@@ -102,24 +103,28 @@ public void onComplete(@NonNull Task<QuerySnapshot> task) {
                 });
     }
 
-    public void logout(View view) {
+    public void logout(View view)
+    {
         Intent intent = new Intent(this, UserLoginActivity.class);
         intent.putExtra("Logout", 0);
         startActivity(intent);
     }
 
```

Comment:
u didnt change some important things, u had to leave a general comment about WIP

Reply:
well, you are right. next time